test/test_utils.py

```diff
@@ -286,6 +286,20 @@ def test_get_authenticator_from_credential_file():
     assert authenticator.token_manager.scope is None
     del os.environ['IBM_CREDENTIALS_FILE']
 
+    file_path = os.path.join(os.path.dirname(__file__), '../resources/ibm-credentials-iam-assume.env')
+    os.environ['IBM_CREDENTIALS_FILE'] = file_path
+    authenticator = get_authenticator_from_environment('service 1')
```

Comment:
"I may not always live on the edge, but when I do... I use spaces in my service names..."  :joy:

Reply:
LOL! We have a few more places where we use service names with spaces in it, but those were also me (😂) and a former contributor. I guess this also covers a scenario so basically a test improvement, hah!